src/ncp-spinel/SpinelNCPInstance.cpp

```diff
@@ -2650,6 +2654,17 @@ SpinelNCPInstance::property_set_value(
 				.finish()
 			);
 
+		} else if (strcaseequal(key.c_str(), kWPANTUNDProperty_ThreadChildTimeout)) {
+			uint32_t child_timeout = any_to_int(value);
+
+			start_new_task(SpinelNCPTaskSendCommand::Factory(this)
+				.set_callback(cb)
+				.add_command(
+					SpinelPackData(SPINEL_FRAME_PACK_CMD_PROP_VALUE_SET(SPINEL_DATATYPE_UINT32_S), SPINEL_PROP_THREAD_CHILD_TIMEOUT, child_timeout)
+				)
+				.finish()
+			);
+
```

Comment:
Let's also save this property in `mSettings` map, this way the last user set value for "ChildTimeout" will be remembered by `wpantund` and if NCP gets reset, wpantund will apply the value back on NCP. 

```
		} else if (strcaseequal(key.c_str(), kWPANTUNDProperty_ThreadChildTimeout)) {
			uint32_t child_timeout = any_to_int(value);
			Data command = SpinelPackData(
				SPINEL_FRAME_PACK_CMD_PROP_VALUE_SET(SPINEL_DATATYPE_UINT32_S),
				SPINEL_PROP_THREAD_CHILD_TIMEOUT,
				child_timeout
			);

			mSettings[kWPANTUNDProperty_ThreadChildTimeout] = SettingsEntry(command);

 			start_new_task(SpinelNCPTaskSendCommand::Factory(this)
				.set_callback(cb)
				.add_command(command)
				.finish()
			);
```